web/cobrands/greenwich/layout.scss

```diff
@@ -257,3 +374,156 @@ body.twothirdswidthpage .content .sticky-sidebar {
         height: 0;
     }
 }
+
+.footer-skyline {
+    background-image: url(https://www.royalgreenwich.gov.uk/site/images/skyline-grey.svg);
+    background-position: bottom center;
+    background-repeat: repeat no-repeat;
+    background-size: contain;
+    -ms-flex: 0 0 auto;
```

Comment:
I’d probably use our `flex()` mixin from `_mixins.scss` for this, rather than rolling your own. Yes it’s half a dozen more lines, but at least then, when we want to drop support for old browsers, we only have drop it from one place.

Reply:
You also use `flex:` on its own a few lines down from here, so I’d make a decision – if you feel backwards compatibility is needed, use the `flex()` mixin everywhere, if it’s not, just use `flex:`.